pkg/utils/utils.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/containers/image/pkg/sysregistries"
+	"github.com/containers/image/signature"
+	"github.com/sirupsen/logrus"
+)
+
+// CheckConfigFiles log messages if one or more files to set up buildah do not exist
+func CheckConfigFiles() {
+	registriesConfPath := sysregistries.RegistriesConfPath(nil)
+	_, err := os.Stat(registriesConfPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Errorf("Required file registries.conf not found in %s, %s", registriesConfPath, err)
+		} else {
+			logrus.Errorf("error checking registries.conf in %s, %s", registriesConfPath, err)
+		}
+	}
+
+	mountsDefaultFilePath := "/usr/share/containers/mounts.conf"
```

Comment:
These	

DefaultMountsFile = "/usr/share/containers/mounts.conf"
OverrideMountsFile = "/etc/containers/mounts.conf"
UserOverrideMountsFile = filepath.Join(os.Getenv("HOME"), ".config/containers/mounts.conf")

 are defined in 
libpod/pkg/secrets/secrets.go

The constants should be used from there.  Also we need to handle the rootless case.  Do things fail if this file does not exists?  If not I say we just ignore it.

Reply:
Better yet, why not put this function into secrets.go and do the check there.  I would also like to see secrets.go moved to buildah so we could stop vendoring libpod into buildah.

Reply:
I think those commands do not fail if there's no mounts file, I'll ignore it...